.rubocop.yml

```diff
@@ -0,0 +1,84 @@
+AllCops:
+  Include:
+    - '**/Rakefile'
+  Exclude:
+    - 'bin/stubs/*'
+    - 'cookbooks/**/*'
+    - 'db/schema.rb'
+    - 'vendor/**/*'
+  TargetRubyVersion: 2.1
```

Comment:
Short answer: we support Ruby >= 2.1 so that's why this is our target.

As for what it does:

> Some checks are dependent on the version of the Ruby interpreter which the inspected code must run on. For example, using Ruby 2.0+ keyword arguments rather than an options hash can help make your code shorter and more expressive... unless it must run on Ruby 1.9.

https://github.com/bbatsov/rubocop#setting-the-target-ruby-version